src/main/resources/data/minecoprocessors/books/manual.txt

```diff
@@ -1,130 +1,291 @@
+§1SAL a§r Arithmetic Left
+§1ROR a§r Rotate Right
+§1ROL a§r Rotate Right
+
+
+
+
+
+§o(Results stored in §1a§r§o).§r
+~~~
+§nFlow Control Instr.§r
+
+§1CMP a, b§r Compare
+§1JMP label§r Jump
+§1JZ label§r Jump if Zero
+§1JNZ label§r not Zero
+§1JC label§r if Carry
+§1JNC label§r if no Carry
+§1DJNZ a, label§r Dec and Jump if not zero
+§1CALL label§r Call
+§1RET§r Return
+~~~
+§nFlags§r
+
+§1Z§r Zero Flag
+§1C§r Carry Flag
+§1F§r Fault Flag
+§1S§r Sleep Flag
+
+
+
 
-  Minecoprocessors
 
-       Redstone
-      Processor
-        Manual
 
 ~~~
-§1Visit the Wiki§r
 
-For more information, visit the wiki:
 
-https://github.com/ToroCraft/Minecoprocessors/wiki
+
+     §o§lIntroduction§r
+
+~~~
+§nSyntax§r
+
+In assembly language we do one operation per line. Each line starts with an instruction, followed by arguments, e.g.:
+
+ §1MOV a, 10§r
+
+assigns 10 to the register §1a§r ("§1a = 10§r").
+~~~
+Instructions which calculate something normally store the result back in the first argument, hence,
+
+ §1MOV a, 12§r
+ §1MOV b, 10§r
+ §1SUB a, b§r
+
+will result in register §1a§r containing §12§r as present value.
+
+~~~
+Comments start with a semicolon:
+
+ §1; This is a comment§r
+
+Labels allow to name program addresses. They are used for conditions, loops, or functions:
+
+§4start:§r
+ §1JMP§r §4start§r
+~~~
+Numbers can be hex, decimal, octal, as well as binary. A byte with all bits set (§11§r) can be written as:
+
+Hexadecimal: §10xff§r
+Decimal: §1-1§r
+Octal: §10o377§r
+Binary: §111111111b§r
+
+The decimal value above is equal to §1-1§r because the byte is ...
+~~~
+interpreted as signed value. That means the highest bit §110000000b§r adds §1-128§r to all other bits, which add §11§r, §12§r, §14§r, §18§r, §116§r, §132§r, and §164§r to the value.
+
+In most cases, binary or hexadecimal representations are best, as we are often dealing with the bits in registers.
+~~~
+§nCore Registers§r
+
+The CPU has four one byte registers §1a, b, c,§r and §1d§r to store values and calculate.
+
+These "variables" do not change any of the signals outside the CPU - they are only internal.
 ~~~
-§1Instructions§r
+§nPorts and Port Config§r
 
-§lMOV a, b§r Move
-§lPUSH a§r Push to Stack
-§lPOP a§r Pop from Stack
-§lNOP§r No Operation
-§lHLT§r Halt Processor
-§lCLC§r Clear Carry
-§lSEC§r Set Carry
-§lCLZ§r Clear Zero
-§lSEZ§r Set Zero
-§lHLT§r Halt Processor
-§lWFE§r Sleep (experimental)
+The CPU interacts with the world around via special registers. These are mapped to the four sides of the device, and we need to tell the CPU if we like to read signals from a side, or want to output redstone power there.
+
+Additionally, we can
 ~~~
-§1Arithmetic Instructions§r
+specify for each I/O if we are interested in the numeric "strength" value of the signal, or only the on/off data.
+§oAlso known as §1analog§r and §1digital§r mode.§r
+
+Lastly, ports can also be used to reset the controller by leaving a side unconfigured or "unconfiguring" it explicitly.
```

Comment:
This seems to at least gloss over or maybe even misinterpret how reset ports are shown to work on the Wiki:
Ports are used as reset ports only if the corresponding bit in the high nybble of PORTS is set, and the port is currently set to input mode, according to [this](https://github.com/ToroCraft/Minecoprocessors/wiki/I-O-Ports) page on the wiki

Reply:
Looks like this is mentioned in lines +196 and +198. Maybe change the wording here to say something like
`Lastly, any of the ports can also be used as a reset for the processor by setting the correct bits in the ports register`